Common.Logging.NLog45/Common.Logging.NLog45.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why is this a NET.Sdk project? It is on purpose? I remember talkin' bout this project being intended only for .netfx/mono scenarios, as on .netcore we will not be using common.logging.

Reply:
As we have just discussed, the idea started out that it would be 'multi-target' in frameworks, and finally we have decided that all projects will be of sdk type, including the Tests project. (But only supporting net45 & net46 except sdk test proyect that will be only net45)